geonode_mapstore_client/client/js/api/geonode/adapter/index.js

```diff
@@ -43,7 +43,7 @@ export const creatMapStoreMap = (body = {}) => {
 */
 export const updateMapStoreMap = (id, body = {}) => {
     const { endpointAdapter = '/mapstore/rest' } = getConfigProp('geoNodeApi') || {};
-    return axios.patch(parseDevHostname(`${endpointAdapter}/resources/${id}/`),
+    return axios.put(parseDevHostname(`${endpointAdapter}/resources/${id}/`),
```

Comment:
Why do we need to change this to put?

Reply:
So Django sets a header for content-type to `text/plain, */*` which causes the request to fail for `patch`. I tried to see how to fix and the only way I could get is to change the http method to `put`.
<img width="642" alt="Screenshot 2021-04-27 at 14 16 03" src="https://user-images.githubusercontent.com/39124174/116232870-571c3780-a763-11eb-9770-b115c5dc8ebd.png">
<img width="680" alt="Screenshot 2021-04-27 at 14 16 10" src="https://user-images.githubusercontent.com/39124174/116232881-5aafbe80-a763-11eb-9eb4-d95d2b1e0df8.png">

Reply:
Okay

Reply:
it turns out there was a problem in my config. I have restored the patch method